rocm_agent_enumerator

```diff
@@ -81,7 +81,7 @@ def staticVars(**kwargs):
     return func
   return deco
 
-@staticVars(search_term=re.compile("gfx[0-9a-fA-F]+"))
+@staticVars(search_term=re.compile(r"gfx[0-9a-fA-F]+"))
```

Comment:
There is no practical difference between using a raw string and a non-raw string, as there are no backslash escape characters involved here.